### ISSUE TITLE
Is there a way to calculate attributes on search?

### ISSUE THREAD

**Issue by YiFanChen99** [zadam/trilium#1198]
Hi all,

I have some notes with attributes like this:
![image](https://i.imgur.com/1pOCD4O.jpg)

Is there a way to search like `(@dateCompleted+@dayPeriod+@daySnoozed)>TODAY` ?

**Comment by zadam** [comment]
No, calculation as part of search string is not supported and isn't for the foreseeable future.

Part of the problem is that all label types are actually strings under the hood. I think the way forward is the script backed saved search as mentioned in #1199
